Project.toml

```diff
@@ -17,19 +17,14 @@ Preferences = "21216c6a-2e73-6563-6e65-726566657250"
 RecursiveFactorization = "f2c3362d-daeb-58d1-803e-2bc74f2840b4"
 Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
 SciMLBase = "0bca4576-84f4-4d90-8ffe-ffa030f20462"
+SciMLOperators = "c0aeaf25-5076-4817-a8d5-81caf7dfa961"
 Setfield = "efcf1570-3423-57d1-acb7-fd33fddbac46"
 SnoopPrecompile = "66db9d55-30c0-4569-8b51-7e840670fc0c"
 SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 Sparspak = "e56a9233-b9d6-4f03-8d0f-1825330902ac"
 SuiteSparse = "4607b0f0-06f3-5cda-b6b1-a6196a1729e9"
 UnPack = "3a884ed6-31ef-47d7-9d2a-63182c4928ed"
 
-[weakdeps]
-HYPRE = "b5ffcf37-a2bd-41ab-a3da-4bd9bc8ad771"
-
-[extensions]
-LinearSolveHYPRE = "HYPRE"
-
```

Comment:
Unrelated change (and below). Better to keep it in the 1.9 sorted way.

Reply:
`Pkg` must've done that. Lemme fix it

Reply:
done